canopen/objectdictionary/__init__.py

```diff
@@ -108,8 +108,8 @@ def __getitem__(
             if isinstance(index, str) and '.' in index:
                 idx, sub = index.split('.', maxsplit=1)
                 return self[idx][sub]
-            name = "0x%X" % index if isinstance(index, int) else index
-            raise KeyError("%s was not found in Object Dictionary" % name)
+            name = f"0x{index:X}" if isinstance(index, int) else index
```

Comment:
Should be padded to four digits?

Don't we already have an attribute for this format? (Sorry, reviewing on mobile, don't have much context right now).

Reply:
It sits in a context where we _don't_ have the object, so no attribute I'm afraid. But we can change this to `:04X` as discussed in other issues.

Reply:
Yes, please.